thread-manager/src/rayon_runtime.rs

```diff
@@ -0,0 +1,82 @@
+use {
+    crate::policy::{apply_policy, parse_policy, CoreAllocation},
+    anyhow::Ok,
+    serde::{Deserialize, Serialize},
+    std::{
+        ops::Deref,
+        sync::{Arc, Mutex},
+    },
+};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(default)]
+pub struct RayonConfig {
+    pub worker_threads: usize,
+    /// Priority in range 1..99
+    pub priority: u8,
+    pub policy: String,
+    pub stack_size_bytes: usize,
+    pub core_allocation: CoreAllocation,
+}
+
+impl Default for RayonConfig {
+    fn default() -> Self {
+        Self {
+            core_allocation: CoreAllocation::OsDefault,
+            worker_threads: 16,
+            priority: crate::policy::DEFAULT_PRIORITY,
```

Comment:
doc-comemnt on `priority` field says range 1..99, but the default here is 0. I think the comment should be adjusted.

Reply:
Yes, it should be from 0 to 99, fixed.